### DISCUSSION TITLE
X6后续更新与维护计划

### DISCUSSION THREAD

**Discussion by BrightXW** [antvis/X6#4387] [Q&A]
请问下，X6接下来还会继续更新维护吗？
看到issues里，很多问题，已经很久都没有修复了

**Comment by zlecheng** [comment]
> 请问下，X6接下来还会继续更新维护吗？ 看到issues里，很多问题，已经很久都没有修复了

不知道，反正最近提的issue基本没人解决和回复了

**Comment by chuyuandu** [comment]
新项目还在纠结要不要用呢，看了下发布日志，最新版本还是一月份的...

**Comment by liangshuai** [comment]
等一个已合并的PR等了七个月， 期待早点发新版本

**Comment by liaodalin19903** [comment]
希望AntV团队能重视X6项目，让维护者更加频繁参与项目的更新。
也希望能多活跃让很多开发者参与项目的贡献。

**Comment by wb-wenbei** [comment]
+1
